.gitignore

```diff
@@ -19,6 +19,7 @@
 /node_modules
 /sent_emails
 /static
+/dev_venv
```

Comment:
Nit-picky on my part, but usually the virtual environment should be in the parent directory.  The "dev_venv" I have in the readme is an arbitrary name, so not really worth adding to the gitignore in my opinion.

Reply:
I often have the venv inside the project because I might have several projects in the same folder (like dalite and courseflow).  But I agree that gitignoring everyone's named venv isn't practical.  Could I suggest something like `/*venv*/` instead?

Reply:
Good call, I'll change the .gitignore